actions/retest/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.16
+
+COPY . /home/src
+WORKDIR /home/src
+
+RUN go build -mod=vendor -o /bin/retest ./main.go
```

Comment:
Building and using the container that contains all sources is not very clean. Consider using a multi-stage build and generating a container (`FROM scratch`) that contains only the `retest` executable.

Reply:
This is only testing locally when the action runs. not sure it makes sense to use multi-stage as we are not pushing image

Reply:
clean examples and best practices are important :smiley: